charts/catalog/templates/apiserver-cert-secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}-apiserver-cert
+  labels:
+    app: {{ template "fullname" . }}-apiserver
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  tls.crt: {{ .Values.apiserver.tls.cert }}
+  tls.key: {{ .Values.apiserver.tls.key }}
+  
```

Comment:
Trailing newline is conventional... and usually not shown by github. What's unusual here is that there were spaces includes on the trailing newline. I will fix that.